doc/contributing/technical-priorities.md

```diff
@@ -160,3 +160,18 @@ and integration with other systems.
 As TypeScript usage continues to grow and gains more prominence in the
 ecosystem, enhancing its support is essential for delivering an improved
 developer experience for newcomers and experienced users alike.
+
+## Non-goals
+
+The following are not considered technical priorities for the project:
+
+* **Unbundling `npm`**. When `npm` was included in the Node.js distribution, it
+  was the only JavaScript package manager available, and it was provided as a
+  way to help developers easily install other JavaScript software. `npm` is also
+  the reference implementation for the `npm` registry, which is the de facto
```

Comment:
nit: I suggest we use "`npm`" to refer to the binary (the npm client), and "npm" otherwise
```suggestion
  the reference implementation for the npm registry, which is the de facto
```